style.css

```diff
@@ -842,11 +796,12 @@ ul {
   border-color: #c1c7d0;
 }
 
-.submission :hover:enabled {
+/* .submission :hover:enabled {
   background: #6070ff;
   color: white;
   box-shadow: calc(0px + 0px) 8px 16px rgb(64 83 252 / 24%);
-}
+  border-color: #c1c7d0;
+} */
```

Comment:
Great work team 🤝

- [ ] Kindly remove all commented code from your style sheet as this is not regarded as a good practice.